### DISCUSSION TITLE
How do I enable this plugin?

### DISCUSSION THREAD

**Discussion by idelice** [Bekaboo/dropbar.nvim#142] [General]
I'm using LazyVim distro, and have installed the plugin. I can confirm it's loading using `lazy` command, but I'm not seeing any winbar popping up on the top bar. 

How does this work?

**Comment by Bekaboo** [comment]
You need to install corresponding language server or treesitter parser & queries for the language you work with.

**Reply by idelice** [reply]
I already have all that for java, typescript, javascript and lua both i still see no winbar appear anywhere.

**Reply by OBLIXi-ON** [reply]
It's not lazy loading at all. I had to disable it in `init.lua` with `lazy = false,`.
